packages/compass-e2e-tests/tests/atlas-login.test.ts

```diff
@@ -213,9 +213,10 @@ describe('Atlas Login', function () {
       await browser.clickVisible(Selectors.LogInWithAtlasButton);
 
       const errorToast = browser.$(Selectors.AtlasLoginErrorToast);
+      await errorToast.waitForDisplayed();
```

Comment:
We forgot to wait for the toast to display but subtle timings seemed to change where it didn't matter before and now it does.